### DISCUSSION TITLE
How many simultaneous users can JsPsych handle?

### DISCUSSION THREAD

**Discussion by SomeUser1411** [jspsych/jsPsych#2796] [Q&A] [answered]
I want to run an experiment simultaneously for 400 subjects, so the question is, can jsPsych handle all those users and create the csv files simultaneously for them?
I am using XHR request to generate csv files using PHP.

**Comment by jodeleeuw** [comment, marked as answer]
Hi @SomeUser1411,

This will depend on the server that you have running the experiment, but it should be OK. The jsPsych code will be running on each participant's computer, so there will be no load on the server during the experiment. The only load will be when folks are downloading the experiment files at the start and then when submitting the data at the end. Since folks will probably be finishing at slightly different times, even if you are running 400 people at once, the load will be staggered and shouldn't create any issues. 400 simultaneous requests isn't an issue for most servers anyways.
